tempesta_fw/sync_socket.h

```diff
@@ -35,11 +35,23 @@ typedef struct ss_proto_t {
 	int			type;
 } SsProto;
 
-/* Flag bits offset for SsProto field. */
-#define	__Flag_Bits		0x10
+/*
+ * Flag bits definition for SsProto.type field.
+ * NOTE: There are also flags definition for this
+ * field in Connection layer (in connection.h)
+ */
+enum {
+	/* Flag bits offset for SsProto field. */
+	__Flag_Bits		= 0x10,
 
-/* No requests longer accepted (flag only for client connections) */
-#define	Conn_Suspected		(0x1 << __Flag_Bits)
+	/*
+	 * Connection is in special state: it is alive and
+	 * continue send responses to client, but no new
+	 * requests longer accepted (flag is intended
+	 * only for client connections).
+	 */
+	Conn_OnHold		= 0x1 << __Flag_Bits,
```

Comment:
Well, I commented Aleksey's comment https://github.com/tempesta-tech/tempesta/pull/848#discussion_r144702374 and thought that it's enough and you just leave `suspected` name untouched. `OnHold` is bad name, at least because we already have `tfw_http_conn_on_hold()` which has completely different meaning. Moreover, the whole #658 change is about differentiation errors from security events (attacks), so all the related stuff must clearly define difference between the events instead of using common names.

Please revert the change.

Reply:
Corrected.